consumer/conf/consumer/consumer.json

```diff
@@ -0,0 +1,7 @@
+{
+    "title": "CKAN portal",
+    "url": "http://ckan:5000",
+    "api_key": "49d4cd30-85ab-467f-8710-676f0ce55df1",
+    "autoconfig_datasets" : true,
```

Comment:
We can kill autoconfig totally.

A adding a subscription to `topics : "*"` is the same as what the old autoconfig montior would handle.

Reply:
`autoconfig`  in the current implementation creates a 1:1 topics:datasets, won't this be the default option if a target dataset isn't specified in `target_options`? Making a mandatory `target_options` will mean the user has to have setup datasets on ckan before hand.

Reply:
Now, topics will be added to the specified target dataset as resources.